ui/src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -169,6 +170,7 @@ class UnconnectedTemplatesIndex extends Component<Props> {
       )
     } catch (err) {
       this.props.notify(communityTemplateUnsupportedFormatError())
+      reportError(err, {name: 'The community template install failed'})
```

Comment:
let's tweak this name to be a little more specific - we already have one that says the install failed.

```suggestion
      reportError(err, {name: 'The community template getTemplateDetails failed'})
```